src/main/java/frc/robot/subsystems/Drivetrain.java

```diff
@@ -12,13 +12,31 @@
 import edu.wpi.first.wpilibj2.command.button.CommandXboxController;
 import frc.robot.Constants.ControlConstants;
 import frc.robot.Constants.OperatorConstants;
+import com.revrobotics.CANSparkMax;
+import com.revrobotics.CANSparkMaxLowLevel.MotorType;
 
-public class Drivetrain extends SubsystemBase {
+import edu.wpi.first.wpilibj.motorcontrol.MotorController;
+import edu.wpi.first.wpilibj2.command.SubsystemBase;
+import edu.wpi.first.wpilibj2.command.button.CommandXboxController;
+import frc.robot.Constants.ControlConstants;
+import frc.robot.Constants.OperatorConstants;
+import frc.robot.subsystems.Drivetrain;
+import frc.robot.RobotContainer;
+
+import edu.wpi.first.wpilibj.TimedRobot;
+import edu.wpi.first.wpilibj.smartdashboard.SmartDashboard;
 
-	private MotorController frontLeft;
-	private MotorController frontRight;
-	private MotorController backLeft;
-	private MotorController backRight;
+import com.revrobotics.RelativeEncoder;
+public class Drivetrain extends SubsystemBase {
+	public CANSparkMax frontLeft;
+	public CANSparkMax frontRight;
+	public CANSparkMax backLeft;
+	public CANSparkMax backRight;
+    public RelativeEncoder e;
+	private MotorController frontLeftController;
+	private MotorController frontRightController;
```

Comment:
Separate `MotorController` objects are not needed! The CANSparkMax object can both control a motor with `.set()` and return that controller's input encoder with `.getEncoder()`